examples/table/table/app.py

```diff
@@ -72,7 +81,7 @@ def startup(self):
         self.table1 = toga.Table(
             headings=headings,
             data=bee_movies[:4],
-            style=Pack(flex=1, padding_right=5),
+            style=Pack(flex=1, padding_right=5, height=250),
```

Comment:
This height modifier shouldn't be required; the fact that it is required is a red flag.

Reply:
@freakboy3742 Thank you for your review! 
I think (1) should be solved - I just did not yet apply Asheesh's fix to this branch. 
As for (2), I searched the Internet but did not find any recommendation about table header styles in Android. In Material Design, they recommend using a bold font with no special background color. Would that be acceptable for you?
As for (3), from what I found and know, fixed headers are not the normal way to create a table in Android. I found some posts where they tried hard to implement a fixed header row and it involved using 2 nested tables.
@paulproteus What is your opinion regarding the points (2) and (3) ?